### ISSUE TITLE
preview thumbnails functionality

### ISSUE THREAD

**Issue by ilant** [mwaterfall/MWPhotoBrowser#277]
It would be nice to add functionality for previewing thumbnails, when opening image in full screen mode in iPad, like in Expedia app photogallery

**Comment by plenio** [comment]
yes, loading the thumbnails on ipad is really slow. Are there ways to get that faster?
